src/Uno.UWP/Globalization/NumberFormatting/CurrencyData.cs

```diff
@@ -0,0 +1,356 @@
+﻿#nullable enable
+
+namespace Uno.Globalization.NumberFormatting;
+
+internal class CurrencyData
+{
+	public int DefaultFractionDigits { get; set; }
+	public string CurrencyCode { get; set; } = "";
+	public string Symbol { get; set; } = "";
+	public bool IsSymbolAfterNumber { get; set; }
+
+	#region Initialize
+	public static readonly CurrencyData Empty = new CurrencyData();
+
+	private static readonly CurrencyData _allCurrencyData = new CurrencyData { CurrencyCode = "ALL", Symbol = "Lek", DefaultFractionDigits = 2 };
```

Comment:
Did you build these fields out of the results from reading what UWP provides?

Reply:
@jeromelaban Hi, I used this [link](https://www.eurochange.co.uk/travel-money/world-currency-abbreviations-symbols-and-codes-travel-money) to grab CurrencyCodes and then used [this code](https://github.com/MohammadHadi2031/UwpNumberFormatting/blob/e7b55d9657abfd32f04d67ea4ea62badd58394e5/DecimalFormatter.Tests/CodeGenerator.cs#L16) to generate fields based on what UWP provides.

Reply:
@jeromelaban I'm using CurrencyIdentifiers as a reference for currency codes.